html_css/project/html-css-foundations-project.md

```diff
@@ -0,0 +1,84 @@
+### Introduction
+
+For this project you'll be creating an entire web page from a design we'll provide for you. If you've been following along you should have the skills you need to accomplish this, but it may not be easy!
+
+The design we're providing you comes in the form of 2 images, one of them an image of the complete website, and one with some details about some of the fonts and colors we've used.
+
+Do _not_ be afraid to use google or go back to previous lessons to look something up. **Real-life, professional developers use google _constantly_ for things that they have been doing for years.** At this point it is not expected that you will have everything memorized so don't worry about it. Additionally, there are a few small details that you may not have encountered in our lessons yet. _This is by design._ These details are minor, and easily searched (i.e. google `css rounded corners`).
+
+Get your project as close as you can to the design but do not worry about getting it pixel-perfect. Don't get out your ruler or count pixels to find the exact margin between the various sections. The point of this assignment is creating something from scratch and getting the various elements in more or less the right position relative to the rest. It doesn't matter if you use `margin: 24px` when the design actually has `margin: 48px`.
+
+_Finally_, feel free to substitute your own content into this design. The images have some meaningless dummy-content, but if you want to make up a business and personalize this page, please feel free to do so! Insert actual images in the placeholders, and feel free to play with the colors and fonts a bit too.
+
+> #### A note about images on the web.
+> You do not have the legal right to use just any image that you find on the web. There are many free images to be found, but make sure that the image you use is actually free for you to use, and make sure to credit the creator of the image in your project. An easy way to give credit is to put the creator's name and contact info in a README file in your repository.
+> Some good places to find free-to-use images on the web include [Pexels](https://www.pexels.com/), [Pixabay](https://pixabay.com/), and [Unsplash](https://unsplash.com/).
+
+### Setting up your project's GitHub repository
```

Comment:
I forgot to add git instructions to the recipe project 🙈

Since the recipe project comes first, would it make sense to have these instructions there and link to them from this project?